plugins/alembic/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+include(${CMAKE_SOURCE_DIR}/cmake/f3dPlugin.cmake)
+
+find_package(Alembic 1.7 REQUIRED)
+
+message(STATUS "Plugin: Alembic ${Alembic_VERSION} found")
+
+f3d_plugin_init()
+
+f3d_plugin_declare_reader(
+  NAME Alembic
+  SCORE 30
```

Comment:
why lower than default ?

Reply:
It only supports a subset of the format right now, so if someone has a custom ABC plugin with the default score, it should take precedence.

Reply:
ok, but I think we can make the some analysis on other readers, eg assimp or occt.

Reply:
I removed it and kept the default score.